file-id/Cargo.toml

```diff
@@ -0,0 +1,24 @@
+[package]
```

Comment:
just to confirm, we don't have to publish this, right ?

Reply:
That's right.
I think there could be some utility in publishing it, but there is also room for improvement.
Currently it only supports the old, low resolution, ids on windows. Newer file systems have file ids with a size 128 bits instead of 64 bits.